src/utils/LoggerUtils.ts

```diff
@@ -578,3 +578,42 @@ export const logEditEventFailedMetadataParsing = (
     ExceptionDetails: exception
   });
 };
+
+export const logFileManagerFileDownloadFailed = (exception: Error): void => {
```

Comment:
nit: maybe rename to logFileManagerDownloadFileFailed to match other log functions to be similar with the log event

Reply:
Need to be updated in the main repo